### PR TITLE
DEVDOCS-4719: [migrate] Widgets API, migrate syntax

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4719]

Migration syntax for Widgets docs

[DEVDOCS-4719]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by bc-traciporter** [comment]
Add copy to widgets-code-samples.md

**Comment by bc-traciporter** [comment]
add copy to the code blocks in widget-tutorial.md

File: docs/api-docs/storefront/widgets/product-widget-tutorial.mdx [hunk 1]
```diff
@@ -252,15 +254,15 @@ The [response](/api-reference/store-management/widgets/widget/getwidget#response
...
+<Callout type="info">
+  The value of the `productId` inside of the `widget_configuration` object is mapped to the `productId` parameter of the `storefront_api_query_params` object.
+</Callout>
 
 ## Place the widget using the API
 
 It is also possible to place widgets programmatically using the API. First, [create a widget](/api-reference/store-management/widgets/widget/createwidget) by sending a `POST` request to `/v3/content/widgets`.
 
-```http
+```http showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/product-widget-tutorial.mdx:265]:
```suggestion
```http showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/product-widget-tutorial.mdx [hunk 2]
```diff
@@ -272,13 +274,11 @@ Accept: application/json
...
 ```
 
-<!-- [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](/api-reference/store-management/widgets/widget/createwidget#requestrunner) -->
-
 Take note of the widget's `uuid` returned in the [response](/api-reference/store-management/widgets/widget/createwidget#responses). You will need the widget's `uuid` to [create a placement](/api-reference/store-management/widgets/placement/createplacement) for your widget.
 
 
 **Response:**
-```json
+```json showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/product-widget-tutorial.mdx:281]:
```suggestion
```json showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/product-widget-tutorial.mdx [hunk 3]
```diff
@@ -289,7 +289,7 @@ Take note of the widget's `uuid` returned in the [response](/api-reference/store
 
 To [create a placement](/api-reference/store-management/widgets/placement/createplacement), send a `POST` request to `/v3/content/placements`.
 
-```http
+```http showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/product-widget-tutorial.mdx:292]:
```suggestion
```http showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/product-widget-tutorial.mdx [hunk 4]
```diff
@@ -303,20 +303,17 @@ Accept: application/json
...
 ```
 
-<!-- [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](/api-reference/store-management/widgets/placement/createplacement#requestrunner) -->
 
 For a list of accepted `template_file` values, see [create a placement](/api-reference/storefront/widgets-api/placement/createplacement).
 
 To [get a list of theme regions](/api-reference/storefront/widgets-api/regions/getcontentregions) for the `region` property, send a `GET` request to `/v3/content/regions?template_file={{TEMPLATE_FILE}}`.
 
-```http
+```http showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/product-widget-tutorial.mdx:311]:
```suggestion
```http showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 5]
```diff
@@ -1,241 +1,241 @@
...
+
+
+
+Below are widget code samples for developers to use as a starting point.
+
+## List
+
+### Template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:11]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 6]
```diff
@@ -1,241 +1,241 @@
...
+<ul>
+    {{#each list_items}}
+    <li style="color:{{color}};">{{text}}</li>
+    {{/each}}
+</ul>
+```
+
+### JSON encoded template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:21]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 7]
```diff
@@ -1,241 +1,241 @@
...
+<ul>\r\n{{#each list_items}}\r\n <li style=\"color:{{color}};\">{{text}}<\/li>\r\n{{\/each}}\r\n<\/ul>
+```
+
+### Configuration
+
+```json showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:27]:
```suggestion
```json showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 8]
```diff
@@ -1,241 +1,241 @@
...
+}
+```
+
+![List](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/widgets-code-samples-01.png "List")
+
+## HTML
+
+### Template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:56]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 9]
```diff
@@ -1,241 +1,241 @@
...
+{{{html}}}
+```
+
+### Configuration
+
+```json showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:62]:
```suggestion
```json showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 10]
```diff
@@ -1,241 +1,241 @@
...
+}
+```
+
+![HTML](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/widgets-code-samples-01.png "HTML")
+
+## Text with styling
+
+### Template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:78]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 11]
```diff
@@ -1,241 +1,241 @@
...
+<style>#bc-simple-text-{{_.id}} {color: {{color}};background: {{background_color}};font-size: {{font_size}};font-style: {{font_style}};font-weight: {{font_weight}};font-family: {{font_family}};text-align: {{text_align}};margin-top: {{margin_top}};margin-bottom: {{margin_bottom}};margin-left: {{margin_left}};margin-right: {{margin_right}};padding-top: {{padding_top}};padding-bottom: {{padding_bottom}};padding-left: {{padding_left}};padding-right: {{padding_right}};}</style><p id='bc-simple-text-{{_.id}}'>{{text}}</p>
+```
+
+### JSON encoded template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:84]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 12]
```diff
@@ -1,241 +1,241 @@
...
+<style>#bc-simple-text-{{_.id}} {color: {{color}};background: {{background_color}};font-size: {{font_size}};font-style: {{font_style}};font-weight: {{font_weight}};font-family: {{font_family}};text-align: {{text_align}};margin-top: {{margin_top}};margin-bottom: {{margin_bottom}};margin-left: {{margin_left}};margin-right: {{margin_right}};padding-top: {{padding_top}};padding-bottom: {{padding_bottom}};padding-left: {{padding_left}};padding-right: {{padding_right}};}<\/style><p id='bc-simple-text-{{_.id}}'>{{text}}<\/p>
+```
+
+### Configuration
+
+```json showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:90]:
```suggestion
```json showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 13]
```diff
@@ -1,241 +1,241 @@
...
+```
+
+
+![Text With Styling](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/widgets-code-samples-01.png "Text With Styling")
+
+## Image slider
+
+### Template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:118]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 14]
```diff
@@ -1,241 +1,241 @@
...
+<ul class="slider">{{#each slides}}<ul class="slide"><img class="mySlides" src="{{image}}"/></ul>{{/each}}</ul><script>var slideIndex=0;carousel();function carousel(){var i;var x=document.getElementsByClassName("mySlides");for (i=0; i < x.length; i++){x[i].style.display="none";}slideIndex++;if (slideIndex > x.length){slideIndex=1}x[slideIndex-1].style.display="block"; setTimeout(carousel, 4000); // Change image every 4 seconds}</script>
+```
+
+### JSON encoded template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:124]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 15]
```diff
@@ -1,241 +1,241 @@
...
+<ul class=\\\"slider\\\">\\n{{#each slides}}\\n<ul class=\\\"slide\\\"><img class=\\\"mySlides\\\" src=\\\"{{image}}\\\"\/><\/ul>\\n{{\/each}}\\n<\/ul>\\n<script>\\nvar slideIndex = 0;\\ncarousel();\\nfunction carousel() {\\nvar i;\\nvar x = document.getElementsByClassName(\\\"mySlides\\\");\\nfor (i = 0; i < x.length; i++) {\\nx[i].style.display = \\\"none\\\"; \\n}\\nslideIndex++;\\nif (slideIndex > x.length) {slideIndex = 1} \\nx[slideIndex-1].style.display = \\\"block\\\"; \\nsetTimeout(carousel, 4000); \/\/ Change image every 4 seconds\\n}\\n<\/script>
+```
+
+### Configuration
+
+```json showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:130]:
```suggestion
```json showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 16]
```diff
@@ -1,241 +1,241 @@
...
+}
+```
+
+![Image Slider](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/widgets-code-samples-04.png "Image Slider")
+
+## Button
+
+### Template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:156]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 17]
```diff
@@ -1,241 +1,241 @@
...
+```
+
+### JSON encoded template
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:162]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-code-samples.mdx [hunk 18]
```diff
@@ -1,241 +1,241 @@
...
+
+```
+
+### Configuration
+
+```json showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-code-samples.mdx:169]:
```suggestion
```json showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx [hunk 19]
```diff
@@ -16,12 +16,17 @@ You can localize your widget template settings by defining translations in the `
...
+### Identify all settings that need a translation 
 
-```json
+Each `label` value must start with the `i18n.` prefix. For example, `"label": "i18n.LineColor"`.
+
+### Prepare your schema translations JSON object
+
+It must contain a default language code along with the other languages you want to support formatted by two lowercase letters, a dash, and at least two alphanumeric characters. For more information, see the [Supported language code schemes](#supported-language-code-schemes) section below.
+
+```json filename="Schema translations example" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx:29]:
```suggestion
```json filename="Schema translations example" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx [hunk 20]
```diff
@@ -56,17 +61,19 @@ The following example demonstrates how to add translations for the widget templa
...
 ```
 
-3. Stringify your data structure. You will pass it as a value to `schema_translations` in the next step.
+### Stringify your data structure
 
-**Stringified schema translations**
+You will pass it as a value to `schema_translations` in the next step.
 
-```json
+```json filename="Stringified schema translations" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx:68]:
```suggestion
```json filename="Stringified schema translations" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx [hunk 21]
```diff
@@ -56,17 +61,19 @@ The following example demonstrates how to add translations for the widget templa
...
 ```
 
-4. Create a widget template by sending a `POST` request to the [Create a Widget Template](/api-reference/store-management/widgets/widget-template/createwidgettemplate) endpoint. The following example creates a simple text widget:
+### Create a widget template 
 
-```http
+To create a widget template, send a `POST` request to the [Create a Widget Template](/api-reference/store-management/widgets/widget-template/createwidgettemplate) endpoint. The following example creates a simple text widget:
+
+```http filename="Example request: Create a widget template" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx:76]:
```suggestion
```http filename="Example request: Create a widget template" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-overview.mdx [hunk 22]
```diff
@@ -30,7 +27,7 @@ Global regions are special regions you can use to place and manage content sitew
 
 To create a global region, add the `--global` suffix to the region name as shown in the following example:
 
-```handlebars
+```handlebars showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-overview.mdx:30]:
```suggestion
```handlebars showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-overview.mdx [hunk 23]
```diff
@@ -50,7 +47,7 @@ The following template examples use the built-in Handlebars [`#each`](https://ha
 
 The list template creates a list where each item in the list can have a different color. The text and color are determined by Handlebars placeholders that are set when creating the [widget](/api-reference/storefront/widgets-api/widget/createwidget).
 
-```json
+```json filename="Example list template" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-overview.mdx:50]:
```suggestion
```json filename="Example list template" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-overview.mdx [hunk 24]
```diff
@@ -61,7 +58,7 @@ The list template creates a list where each item in the list can have a differen
 
 The slider template creates an image slider where each item in the slider can have a different image. The images are determined by Handlebars placeholders that are set when creating the [widget](/api-reference/storefront/widgets-api/widget/createwidget).
 
-```json
+```json filename="Example slider template" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-overview.mdx:61]:
```suggestion
```json filename="Example slider template" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-overview.mdx [hunk 25]
```diff
@@ -80,9 +77,7 @@ A widget configuration is the JSON payload that defines the content you can rend
 
 In the following example, we define the `list_items` array and supply values for the color and text of each list item. Since you create the template separately from the configuration, you can use the same template UUID multiple times.
 
-**Widget configuration list items**
-
-```json
+```json filename="Example widget configuration with list items" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-overview.mdx:80]:
```suggestion
```json filename="Example widget configuration with list items" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-overview.mdx [hunk 26]
```diff
@@ -96,9 +91,7 @@ In the following example, we define the `list_items` array and supply values for
 }
 ```
 
-**Widget configuration slides**
-
-```json
+```json filename="Example widget configuration with slides" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-overview.mdx:94]:
```suggestion
```json filename="Example widget configuration with slides" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 27]
```diff
@@ -1,327 +1,313 @@
...
+
+## Create a region
+
+For a widget to appear on a store's page, you have to place it in a region. Regions are added and removed at the file level by editing a page's template. 
+
+Let's start by adding a new region called `category_header_banner` to your store's category page template. You will use this region to position your widget later.
+
+In `templates/pages/category.html`, add a new region `{{{region name="category_header_banner"}}}` below the page heading.
+
+```html showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:25]:
```suggestion
```html showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 28]
```diff
@@ -1,327 +1,313 @@
...
+  * To edit theme files locally, use [Stencil CLI](/stencil-docs/installing-stencil-cli/installing-stencil).
+  * To edit theme files in the control panel, use [Page Builder](https://support.bigcommerce.com/s/article/Stencil-Themes#edit).
+</Callout>
+
+
+### Verify region creation
+
+To verify region creation, send a `GET` request to [`/v3/content/regions?template_file=pages/category`](/api-reference/storefront/widgets-api/regions/getcontentregions). Make sure to specify the  `template_file=pages/category` query string parameter to get the category template's regions.
+
+```http filename="Example request: Get theme regions" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:49]:
```suggestion
```http filename="Example request: Get theme regions" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 29]
```diff
@@ -1,327 +1,313 @@
...
+GET /stores/{{STORE_HASH}}/v3/content/regions?template_file=pages/category
+Host: api.bigcommerce.com
+X-Auth-Token: {{ACCESS_TOKEN}}
+Accept: application/json
+```
+
+Look for the region's name in the response.
+
+```json filename="Example response: Get theme regions" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:58]:
```suggestion
```json filename="Example response: Get theme regions" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 30]
```diff
@@ -1,327 +1,313 @@
...
+  ],
+  "meta": {}
+}
+```
+
+## Create a widget template
+
+Widgets derive from widget templates. Before you can create a widget, you must first create its template. To do so, send a `POST` request to [`/v3/content/widget-templates`](/api-reference/storefront/widgets-api/widget-template/createwidgettemplate).
+
+```http filename="Example request: Create a widget template" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:79]:
```suggestion
```http filename="Example request: Create a widget template" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 31]
```diff
@@ -1,327 +1,313 @@
...
+
+{
+  "name": "Header Images",
+  "template": "{{#each images}}<a href='{{image_url}}'><img src={{image_source}} style='width:33.3%'/></a>{{/each}}"
+}
+```
+
+**Response:**
+
+```json filename="Example response: Create a widget template" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:94]:
```suggestion
```json filename="Example response: Create a widget template" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 32]
```diff
@@ -1,327 +1,313 @@
...
+* Multiple widgets can use the same widget template.
+</Callout>
+
+
+## Create a widget
+
+To create a widget, use the widget template `uuid` from the previous step. Send a `POST` request to [`/v3/content/widgets`](/api-reference/storefront/widgets-api/widget/createwidget) making sure to replace the `widget_template_uuid` placeholder value with your template's `uuid`.
+
+
+```http filename="Example request: Create a widget" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:124]:
```suggestion
```http filename="Example request: Create a widget" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 33]
```diff
@@ -1,327 +1,313 @@
...
+</Callout>
+
+
+## Create a placement
+
+In the control panel UI, users can drag and drop widgets to place them in a region on a page. For this tutorial, we will use the Widgets API to place the widget programmatically. 
+
+To place your widget in the `category_header_banner` region of a category page, send a `POST` request to [`/v3/content/placements`](/api-reference/storefront/widgets-api/placement/createplacement).
+
+```http filename="Example request: Create a widget placement" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:171]:
```suggestion
```http filename="Example request: Create a widget placement" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 34]
```diff
@@ -1,327 +1,313 @@
...
+
+1. In the `/templates/pages` folder, create a `/custom/category` folder and add a `custom-category.html` file.
+
+![Custom Template 01](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/widgets-custom-template-01.png "Custom Template 01")
+
+2. Add template content and a new region. (You can copy the content from `category.html`.)
+
+3. Open the `config.stencil.json` file and update the `custom-category.html` property. The URL you define in `config.stencil.json` will be used for category mapping.
+
+```json filename="Example custom template" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:213]:
```suggestion
```json filename="Example custom template" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 35]
```diff
@@ -1,327 +1,313 @@
...
+  "normalStoreUrl": "{STORE URL}",
+  "port": "3000"
+}
+```
+
+If using Stencil CLI, push and apply your changes.
+
+4. To create a new category using the [Catalog API](/api-reference/store-management/catalog), send a `POST` request to [`/v3/catalog/categories`](/api-reference/store-management/catalog/category/createcategory). Use the URL defined in the `config.stencil.json` category mapping. In our example, it is `/custom-widget-templates/`.
+
+```http filename="Example request: Create a category" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:232]:
```suggestion
```http filename="Example request: Create a category" showLineNumbers copy
```

File: docs/api-docs/storefront/widgets/widgets-tutorial.mdx [hunk 36]
```diff
@@ -1,327 +1,313 @@
...
+
+5. You can now [create a placement](#create-a-placement) for the widget you created in the previous steps. 
+
+## Create a user interface
+
+BigCommerce's [Page Builder](/stencil-docs/page-builder/page-builder-overview) tool allows you to customize different style elements of your theme in the store's control panel. You can use Page Builder to configure and place widgets onto pages to tailor your storefront. To make your widget compatible with Page Builder, add custom configuration settings to the widget template by including the `schema` property in the [create widget template](#create-a-widget-template) request. 
+
+The following is an example of a widget template compatible with Page Builder.
+
+```json filename="Example widget template" showLineNumbers
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/widgets/widgets-tutorial.mdx:262]:
```suggestion
```json filename="Example widget template" showLineNumbers copy
```

**Review by bc-traciporter — COMMENTED** [review]
left comments

File: docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx [hunk 37]
```diff
@@ -16,12 +16,17 @@ You can localize your widget template settings by defining translations in the `
 
 The following example demonstrates how to add translations for the widget template settings using the `schema_translations` property. 
 
-1. Identify all of the settings that need a translation. Each `label` value must start with the `i18n.` prefix. For example, `"label": "i18n.LineColor"`.
-2. Prepare your schema translations JSON object. It must contain a default language code along with the other languages you want to support formatted by two lowercase letters, a dash, and at least two alphanumeric characters. For more information, see the [Supported language code schemes](#supported-language-code-schemes) section below.
+<Steps>
```
[the remark this answers is not in the document]
Comment (bc-andreadao) [reply docs/api-docs/storefront/widgets/widgets-localizing-settings.mdx]: ✔️

**Review by bc-traciporter — APPROVED** [review]
All files look good

**Review by slsriehl — APPROVED** [review]

**Pull request merged**